packages/react-reconciler/src/__tests__/ReactPure-test.internal.js

```diff
@@ -35,14 +35,14 @@ describe('pure', () => {
   }
 
   // Tests should run against both the lazy and non-lazy versions of `pure`.
-  // To make the tests work for both versions, we wrap the non-lazy verion in
+  // To make the tests work for both versions, we wrap the non-lazy version in
   // a lazy function component.
   sharedTests('normal', (...args) => {
     const Pure = React.pure(...args);
-    function Indirection(props) {
-      return <Pure {...props} />;
+    function Indirection(props, ref) {
+      return <Pure {...props} ref={ref} />;
     }
-    return Promise.resolve(Indirection);
+    return Promise.resolve(React.forwardRef(Indirection));
```

Comment:
Ironically had to figure out how to compose these for the sole purpose of testing this change.